village/src/App.js

```diff
@@ -1,24 +1,74 @@
+// DEPENDANCIES:
 import React, { Component } from 'react';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
 
-import './App.css';
+// COMPONENTS:
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
+import Home from './components/Home';
+import Navigator from './components/Navigator';
+
+// STYLES:
+import './App.css';
 
+// CLASS COMPONENT:
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      smurfs: [],
+      smurfs: []
     };
   }
-  // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
-  // Notice what your map function is looping over and returning inside of Smurfs.
-  // You'll need to make sure you have the right properties on state and pass them down to props.
+
+  // API GET REQUEST:
+  async componentDidMount() {
+    try {
+      const res = await axios.get('http://localhost:3333/smurfs');
+      this.setState({ smurfs: res.data });
+    } catch (err) {
+      console.log(err.message);
+    }
```

Comment:
LOOK AT CHU
Love the use of async/await and the try/catch blocks -> syntactic sugar 🍬 
🔥 🔥